linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+# TODO: The plan is not to need this for RHEL>=8.4
+prodtype: rhel8
+
+title: 'OpenSSL uses strong entropy source'
+
+description: |-
+    To set up an <tt>openssl</tt> wrapper that adds a <tt>-rand /dev/random</tt> option to the <tt>openssl</tt> invocation,
```

Comment:
Before this line I'd say that the wrapper is a way to achieve fulfill this rule. The goal of the rule is not to setup a wrapper, but to make sure OpenSSL is using a strong entropy source.

Raw suggestion: `By default, OpenSSL doesn't always use a SP800-90A compliant random number generator.
A way to configure OpenSSL to always use a strong source is to setup a wrapper that adds ...`